deepchem/feat/sc_rna_seq_featurizers.py

```diff
@@ -0,0 +1,288 @@
+from deepchem.feat import Featurizer
+import numpy as np
+import pandas as pd
+import logging
+from typing import Any, Iterable, Tuple, List
+
+logger = logging.getLogger(__name__)
+
+
+class ACTINNFeaturizer(Featurizer):
+    """
+    This class is the implementation of all transformations performed to
```

Comment:
More detail; which transformations are these? What do they do? Please add a citation for the scientific context